tests/test_vendor.py

```diff
@@ -9,6 +9,7 @@
         import other
         from other import name1
         from other.name2 import name3
+        import other.name as name1
```

Comment:
To match the substitution regex, this probably needs to be:

```suggestion
        import other.name4 as name5
```